canonical/snapcraft.io
# Add policy endpoints

## Done
- Add the following endpoints
    - `[GET] /admin/store/<store_id>/models/<model_name>/policies`
    - `[POST] /admin/store/<store_id>/models/<model_name>/policies`
## How to QA
- login with `login-beta`
- `GET`:
    - go to https://snapcraft-io-4333.demos.haus/admin/store/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/alimot_test_model2/policies
    - go to https://snapcraft-io-4333.demos.haus/admin/store/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/alimot_test_model/policies
- `POST`:
    - the post endpoint should be tested using a front end client with payload {"signing_key": string sha3-384}

## Issue / Card https://warthogs.atlassian.net/browse/WD-4659
Fixes #

## Screenshots
